examples/meetups/app/constants/Meetup.js

```diff
@@ -0,0 +1 @@
+export const apiKey = '2e15524052566e296b4b37c346f6173';
```

Comment:
@grgur I think you shouldn't leave your API key like this or ask users to explicitly add a API key.